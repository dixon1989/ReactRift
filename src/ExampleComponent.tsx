// src/ExampleComponent.js
import React from "react";
import { View, Text } from "react-native";

const ExampleComponent = () => {
  return (
    <View>
      <Text>Hello, Expo Jest Testing!</Text>
    </View>
  );
};

export default ExampleComponent;
