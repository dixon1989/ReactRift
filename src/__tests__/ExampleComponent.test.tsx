import React from "react";
import renderer from "react-test-renderer";
import ExampleComponent from "../ExampleComponent";

describe("ExampleComponent", () => {
  it("renders correctly", () => {
    // Render the component
    const component = renderer.create(<ExampleComponent />);

    // Get the rendered tree
    const tree = component.toJSON();

    // Assert that the component matches the snapshot
    expect(tree).toMatchSnapshot();
  });
});
