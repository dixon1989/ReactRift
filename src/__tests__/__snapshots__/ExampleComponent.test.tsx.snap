// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent renders correctly 1`] = `
<View>
  <Text>
    Hello, Expo Jest Testing!
  </Text>
</View>
`;
